e2e/fs-router.spec.ts

```diff
@@ -81,5 +81,18 @@ for (const mode of ['DEV', 'PRD'] as const) {
       await page.goto(`http://localhost:${port}/_components/Counter`);
       await expect(page.getByText('404 Not Found')).toBeVisible();
     });
+
+    test('alt click', async ({ page }) => {
```

Comment:
@rmarscher This test seems a little bit flaky.
https://github.com/wakujs/waku/actions/runs/14210540846/job/39816803332

Reply:
https://github.com/wakujs/waku/actions/runs/14210920213/job/39817854884
It happens often on macos.

Reply:
https://github.com/wakujs/waku/actions/runs/14218743873/job/39841399899
Happening more than expected.

Reply:
https://github.com/wakujs/waku/pull/1319#issuecomment-2777395732

@tylersayshi

Reply:
This seems worth a try: https://github.com/wakujs/waku/pull/1351

Reply:
I can retry "E2E on macos-latest (Node 22.7.0) - (2/4)" several times there.